functions/completed-checks/v1.js

```diff
@@ -0,0 +1,256 @@
+'use strict'
+
+const moment = require('moment')
+const sqlService = require('less-tedious')
+const uuid = require('uuid/v4')
+const winston = require('winston')
+const { TYPES } = require('tedious')
+
+winston.level = 'error'
```

Comment:
Should be using `context.log` not `winston` in all functions.

Reply:
@jon-shipley removed as it wasn't being used at all anyway.